### ISSUE TITLE
Missing Label - Lack of accessibility 

### ISSUE THREAD

**Issue by KhaderMurtaja** [FACG6/ToDo-Chocolate-Team#35]
Your html tag :
    <input type="text" name="description" value="">
does not have any label to wrap it !

**Comment by AymanAlqoqa** [comment]
thank you for your hint.
we will accomplish this
